tests/test_pipelines_conversational.py

```diff
@@ -295,11 +324,11 @@ def test_integration_torch_conversation_blenderbot_400M(self):
             " Hello! How are you doing today? I just got back from a walk with my dog.",
         )
 
-        conversation_1 = Conversation(" Lasagne   hello")
+        conversation_1 = Conversation("Lasagne   hello")
         result = nlp(conversation_1, encoder_no_repeat_ngram_size=3)
         self.assertEqual(
             result.generated_responses[0],
-            " Lasagne is my favorite Italian dish. Do you like lasagne?",
+            " Do you like lasagne? It is a traditional Italian dish consisting of a shepherd's pie.",
```

Comment:
This is exactly ParlAI answer for this prompt. The difference was linked to the `max_length` problem before this PR.